SYCL/Plugin/level_zero_queue_profiling.cpp

```diff
@@ -0,0 +1,62 @@
+// REQUIRES: gpu, level_zero
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple  %s -o %t.out
+// RUN: env ZE_DEBUG=-1 %GPU_RUN_PLACEHOLDER %t.out 2>&1 | FileCheck --check-prefixes=WITHOUT %s
+// RUN: env ZE_DEBUG=-1 %GPU_RUN_PLACEHOLDER %t.out profile 2>&1 | FileCheck --check-prefixes=WITH %s
+
+// Test case adapted from the SYCL version of Rodinia benchmark hotspot.
+
+// Check the expected output when queue::enable_profiling is not specified
+//
+// WITHOUT: ze_event_pool_desc_t flags set to: 1
+// WITHOUT: terminate called after throwing an instance of
+// 'cl::sycl::runtime_error' WITHOUT:   what():  Native API failed. Native API
```

Comment:
Probably a typo or clang-format messed up these lines. If so, then could you please disable clang-format for these lines.